Gems/ROS2/Code/Source/Camera/ROS2CameraSensorEditorComponent.cpp

```diff
@@ -113,8 +116,8 @@ namespace ROS2
 
         transform.SetUniformScale(frustumScale);
         debugDisplay.DepthTestOff();
-        const float ver = 0.5f * AZStd::tan((AZ::DegToRad(m_VerticalFieldOfViewDeg * 0.5f)));
-        const float hor = m_height * ver / m_width;
+        const float ver = 0.5f * AZStd::tan((AZ::DegToRad(m_cameraSensorConfiguration.verticalFieldOfViewDeg * 0.5f)));
+        const float hor = m_cameraSensorConfiguration.height * ver / m_cameraSensorConfiguration.width;
```

Comment:
nit/standards: Prefer to avoid abbreviations (so this would be `vertical` and `horizontal` (I realize this was like this before so feel free to ignore me, just a super small thing).